freecad/marz/feature/import_svg.py

```diff
@@ -198,8 +199,13 @@ def import_custom_shapes(
     for obj in tmp_doc.Objects:
         if match_ergo_cut(obj, ergo_cuts):
             continue
-        elif obj.Name == 'contour':
-            validation.append(ImportValidationItem('Contour', obj.Name, 'Found'))
+
+        m = NAME_ID_PATTERN.match(obj.Name)
+        if not m:
+            continue
+        name = m.group(1)
+        if name == 'contour':
+            validation.append(ImportValidationItem('Contour', name, 'Found'))
             contour = obj
```

Comment:
All cases need to compare against `name` instead of `obj.Name`

Reply:
bwhaha, true dat!